addons/ammunition/CfgMagazines.hpp

```diff
@@ -1,56 +1,83 @@
 class CfgMagazines {
     class CA_Magazine;
-    class 30Rnd_556x45_Stanag_green;
     class ACE_30Rnd_556x45_Stanag_M995_AP_mag;
     class ACE_30Rnd_556x45_Stanag_Mk262_mag;
     class ACE_30Rnd_556x45_Stanag_Mk318_mag;
 
+    // Renaming for matching name conventions.
+    class 30Rnd_556x45_Stanag: CA_Magazine {
+        displayName = $STR_Ammunition_30Rnd_556x45_Stanag_Display;
```

Comment:
All strings should be quoted.
```suggestion
        displayName = "$STR_Ammunition_30Rnd_556x45_Stanag_Display";
```